docs/HOWTO_INTEGRATE_WITH_ACTIONKIT.md

```diff
@@ -0,0 +1,31 @@
+# How to Integrate with Action Kit
+
+## Step One - making sure your variables are set in your production/development deployment environment
+- ACTION_HANDLERS should equal "actionkit-rsvp"
+- AK_BASEURL should be your base url from your ActionKit account
+- AK_SECRET should be your secret from your ActionKit account
+
+## Step Two - making sure you have the correct data in your contacts list
+
+- In order for ActionKit RSVP Syncing to work with Spoke, we have to make sure we're including the correct information in our CSV upload AND set up the interaction steps correctly in the campaign script.
+
+### Uploading CSVs with the Correct Information
+
+- When you upload a CSV to Spoke, you must make sure you have the following fields filled out for each contact:
+  * external_id (some reference to an id of the user in actionkit)
```

Comment:
it'd be helpful to specify whether this is the akid (usually found in the url) or the actionkit user_id (found in the core_user table in the actionkit database)